pages/sequential.jsx

```diff
@@ -78,9 +79,14 @@ module.exports = React.createClass({
     if (!this.state.showCvcHint) {
       cvcHintClassName += " hidden";
     }
+    // FIXME: Right now we only display this header for `en` locales
+    // this should be change to display for all once we have other locales translated.
+    if (/^(en)(\b|$)/.test(this.props.locales[0])) {
+      header = (<Header alt={this.getIntlMessage('donate_to_mozilla')}/>);
+    }
     return (
       <div className={className}>
-        <Header/>
+        {header}
```

Comment:
Given that the alt isn't displayed to the user, is it safe to say we can remove locale check?